src/akkudoktoreos/server/fastapi_server.py

```diff
@@ -222,29 +261,84 @@ def site_map() -> RedirectResponse:
     return RedirectResponse(url="/docs")
 
 
-@app.get("/", include_in_schema=False)
-def root() -> RedirectResponse:
-    # Redirect the root URL to the site map
-    return RedirectResponse(url="/docs")
+# Keep this last
+# Also keep the single endpoints to assure openai.json is always build the same way.
 
 
-if __name__ == "__main__":
-    try:
-        config.run_setup()
-    except Exception as e:
-        print(f"Failed to initialize: {e}")
-        exit(1)
+@app.delete("/{path:path}")
```

Comment:
what are those used for? if they are unused, should remove them

Reply:
These are the proxy functions that call into `proxy()`.  I had to list them separately to have a consistent generation of openapi.json as mentioned above. They are definitely necessary for the proxy function.

Reply:
Got it, we should however move this functionality to a proper load balancer in the future. And those proxy calls should not be in the server API (include_in_schema=False)

Reply:
> server API (include_in_schema=False)

Das sieht gut aus.